### PR TITLE
Add chip atom

### PR DESCRIPTION
**Pull request by nassimerrahoui**
This is PR add a Tikui component called Chip to modelize one module in the landscape + integration into the webapp. This is related to CSS migration to Tikui.

I'm working with @Gnuk to coordonate CSS migration to Tikui.

- Preparation for [Migrate landscape css to tikui](https://github.com/jhipster/jhipster-lite/issues/3970)
- Related to [Put landscape in light](https://github.com/jhipster/jhipster-lite/pull/4088)
- Related to [UX/UI: improve the CSS](https://github.com/jhipster/jhipster-lite/issues/2894)

Here is what it looks like with the component : 

Capture d’écran 2022-10-31 à 10 22 13

Capture d’écran 2022-10-31 à 09 55 11

Capture d’écran 2022-10-31 à 09 48 15

### CODE REVIEW

**Comment by nassimerrahoui** [comment]
@Gnuk I think it's ready for a first review

**Comment by Gnuk** [comment]
I think you need to add an example of the applied chip inside the pattern library

**Comment by nassimerrahoui** [comment]
> I think you need to add an example of the applied chip inside the pattern library

Done 👍

**Comment by nassimerrahoui** [comment]
@Gnuk I also updated the screenshots

**Review by Gnuk — CHANGES_REQUESTED** [review]

**Comment by Gnuk** [comment]
> @Gnuk I also updated the screenshots

Sorry, I just forgot to apply the review, T-T

File: src/main/style/token/_colors.scss [hunk 1]
```diff
@@ -29,3 +29,43 @@ $jhlite-global-color-link-light: $jhlite-global-color-light !default;
...
 $jhlite-global-color-success: #480 !default;
 $jhlite-global-color-error: #902 !default;
+$jhlite-valid-highlight-animation: valid-highlight-animation 2s infinite;
+$jhlite-invalid-highlight-animation: invalid-highlight-animation 2s infinite;
+$jhlite-valid-highlight-start-color: #3ebf5a;
+$jhlite-valid-highlight-end-color: $jhlite-global-color-success;
+$jhlite-invalid-highlight-start-color: #bf3e3e;
+$jhlite-invalid-highlight-end-color: $jhlite-global-color-error;
+
+@keyframes valid-highlight-animation {
```
[the remark this answers is not in the document]
Comment (nassimerrahoui) [reply src/main/style/token/_colors.scss]: Done 👍

File: src/main/webapp/app/module/primary/landscape/Landscape.vue [hunk 2]
```diff
@@ -152,6 +152,7 @@ $jhipster-landscape-padding: 20px;
 
 .jhipster-landscape-element {
   list-style-type: none;
+  background: #fff;
```
[the remark this answers is not in the document]
Comment (nassimerrahoui) [reply src/main/webapp/app/module/primary/landscape/Landscape.vue]: Found :)

**Comment by nassimerrahoui** [comment]
> > @Gnuk I also updated the screenshots
> 
> Sorry, I just forgot to apply the review, T-T

@Gnuk it's ok 👌

**Comment by pascalgrimaud** [comment]
Adding a bounty on this
Thanks for the work @nassimerrahoui !

**Review by Gnuk — APPROVED** [review]

**Comment by nassimerrahoui** [comment]
> Adding a bounty on this Thanks for the work @nassimerrahoui !

@pascalgrimaud @Gnuk Nice, thank you :D

**Review by Gnuk — APPROVED** [review]

**Comment by pascalgrimaud** [comment]
@nassimerrahoui : approved

**Pull request merged**